pets.json

```
@@ -18,5 +18,15 @@
     "github": "chsienki",
     "instagram": "brotherlucca",
     "twitter": ""
+  },
+  {
+    "name": "Margarito",
+    "jobTitle": "Libraries Engineer",
+    "img": "img/margarito.png",
```

@chsienki if hosting the pic in the `img/` folder does not work, I also uploaded to my personal OneDrive and shared it publicly:

```suggestion
    "img": "https://dsm01pap002files.storage.live.com/y4mEjWbjaXwlFOxIKJE7eYuRKvlqJ24ikhthFE_I8RHYLC9pIijd0QxAyC14YTKLcAeZgDGpXNxVe6vCmhuMYmon23otYMePmupP3eYCvHF7ww_UHCfuM5BD9n_3QOs-FBtldD3Gm8CzEtB3GHh0pSD4ypMtmoBZNyu1aZ1F7LnXI-fhBVWypvhVun7zL_YtQc7?width=760&height=760&cropmode=none",
```